### DISCUSSION TITLE
Transactions in connection management

### DISCUSSION THREAD

**Discussion by popovicieduard** [adonisjs/core#1502] [Help] [answered]
I was wondering what is the behaviour of using a transaction with the dual mode in : https://preview.adonisjs.com/guides/database/connections-management

Is the transaction shared between the 2 connections, or there is no such concept as sharing a transaction between 2 db's ?

**Comment by McSneaky** [comment, marked as answer]
All queries in transaction have to use same connection. It's limit on most of the DBs

**Reply by thetutlage** [reply]
Yes, once a transaction is created. All queries goes to the write replica
